### DISCUSSION TITLE
添加health端点

### DISCUSSION THREAD

**Discussion by IamTaoChen** [lejianwen/rustdesk-api#63] [Ideas]
我在想也许可以添加一个`/health` 端点; 但是不知道该怎么去检查，id 和 relay server

**Comment by IamTaoChen** [comment]
@lejianwen 

既然现在可以是使用服务端命令查看 ID， RELAY server的状态，是否可以添加/healthz端点
* `/healthz` ：显示三者 AND 关系，也许可以设置？API,ID,RELAY？
* `/healthz/api` ：api 是否可用
* `/healthz/id` ：id server是否可用
* `/healthz/relay?{}`：relay server是抖可用。因为可以设置多个relay server。也许有一个办法检查这些？或者至少本地的这个relay server可以

**Reply by IamTaoChen** [reply]
> * 也许有一个办法检查这些？

也许可以尝试建立ws来确定？如果不能建立，那就是服务不健康，如果可以建立（就当是健康的？）。

**Reply by lejianwen** [reply]
在s6镜像中，有health check

```
/package/admin/s6/command/s6-svstat /run/s6-rc/servicedirs/hbbr || exit 1
/package/admin/s6/command/s6-svstat /run/s6-rc/servicedirs/hbbs || exit 1
```

所以感觉没必要自己实现检查
如果要实现，一般就是建立socket看能否成功

或者使用第三方工具，比如 https://github.com/louislam/uptime-kuma
